libs/core/coroutines/include/hpx/coroutines/detail/context_posix.hpp

```diff
@@ -340,7 +342,7 @@ namespace hpx { namespace threads { namespace coroutines {
 #if defined(HPX_HAVE_THREADS_GET_STACK_POINTER)
                 return get_stack_ptr() - reinterpret_cast<std::size_t>(m_stack);
 #else
-                return (std::numeric_limits<std::ptrdiff_t>::max)();
+                return (std::numeric_limits<std::ptrdiff_t>::max) ();
```

Comment:
This just needs reverting.

Reply:
Darn, different versions of clang-format still do different things...

Reply:
Are you using 11? 9 has some questionable formatting choices at least in the sender/receiver files, and I wouldn't mind bumping that up at least to 10 (and I don't mind 11 either).

Reply:
Yes, I'm using V11 here. I'd be fine with bumping the version. This will cause various reformattings, however.

Reply:
Ok, let's look into bumping it, but separately from this PR.